Error: Can't resolve '../../deps/phoenix_live_view/assets/css/live_view.css'

```terminal

       Using default npm version 3.10.9
-----> Building dependencies
       Loading node modules from cache
       Installing node modules
       Caching node modules
       Restoring cached assets
sending incremental file list
created directory priv/static
./
apple-touch-icon-114x114-c49f5d87803b0e6804f447fcadec9530.png
apple-touch-icon-114x114.png
apple-touch-icon-120x120-8ae8a46e79bb3a97b5975488ccac8043.png
apple-touch-icon-120x120.png
apple-touch-icon-144x144-17952478befa501416961b04c34509f3.png
apple-touch-icon-144x144.png
apple-touch-icon-152x152-e3a00893d1ee9d957db6067f449516d4.png
apple-touch-icon-152x152.png
apple-touch-icon-57x57-15c999d24f2304c06dec846308fdf9a7.png
apple-touch-icon-57x57.png
apple-touch-icon-60x60-ef88fd0d8a2ca6b1a4cc48b7d3b4411e.png
apple-touch-icon-60x60.png
apple-touch-icon-72x72-73762f1de8fb3652136d40a78136b57b.png
apple-touch-icon-72x72.png
apple-touch-icon-76x76-674b64ea0b764b2b87ad8aaead3e55ea.png
apple-touch-icon-76x76.png
cache_manifest.json
favicon-128-6d960e0209e81b4c9ba727f13a8df7e6.png
favicon-128.png
favicon-16x16-9293528cfd6ed310b7f1c05199e30413.png
favicon-16x16.png
favicon-196x196-96d3a21a8f11507c5d69feda604054e7.png
favicon-196x196.png
favicon-32x32-52b0fbcf1d3a48a90093def58d03cd1c.png
favicon-32x32.png
favicon-77620264abcdf6b6b02999f31d0c4a54.ico
favicon-96x96-8ac09178afeb7229c8c5e69f58bddde3.png
favicon-96x96.png
favicon.ico
mstile-144x144-17952478befa501416961b04c34509f3.png
mstile-144x144.png
mstile-150x150-835729ce7488b9b7ab53925ba3d83058.png
mstile-150x150.png
mstile-310x150-65b2234a2a6c8b15626dee8e3da85845.png
mstile-310x150.png
mstile-310x310-cf3cf15e23e45c4d2f6e20bd91953a77.png
mstile-310x310.png
mstile-70x70-6d960e0209e81b4c9ba727f13a8df7e6.png
mstile-70x70.png
robots-067185ba27a5d9139b10a759679045bf.txt
robots-067185ba27a5d9139b10a759679045bf.txt.gz
robots.txt
robots.txt.gz
css/
css/active_admin.css-26d0094646fa81252338b7aa03a91bfe.css
css/active_admin.css-26d0094646fa81252338b7aa03a91bfe.css.gz
css/active_admin.css.css
css/active_admin.css.css-7fc207b998f2d6e6274b1f99dc9d0cc9.map
css/active_admin.css.css.gz
css/active_admin.css.css.map
css/admin_lte2-fe9384703a9cabace900ab5fecb46e6f.css
css/admin_lte2-fe9384703a9cabace900ab5fecb46e6f.css.gz
css/admin_lte2.css
css/admin_lte2.css-0403e06bae0f931ee659e70af90fd041.map
css/admin_lte2.css.gz
css/admin_lte2.css.map
css/app-3b1d618b584a418df11f79a338714ce3.css
css/app-3b1d618b584a418df11f79a338714ce3.css.gz
css/app.css
css/app.css.gz
fonts/
fonts/FontAwesome-6246dbecca8e97839faec176527b0320.otf
fonts/FontAwesome.otf
fonts/fontawesome-webfont-6c4c3504e3402b6f1b75332b796da253.svg
fonts/fontawesome-webfont-6c4c3504e3402b6f1b75332b796da253.svg.gz
fonts/fontawesome-webfont-9f5388fac608dfe77f4343d38abf122f.ttf
fonts/fontawesome-webfont-9f5388fac608dfe77f4343d38abf122f.ttf.gz
fonts/fontawesome-webfont-a1a450ea331a3fc89e867ecad963b6b2.woff2
fonts/fontawesome-webfont-a247f4358b253516c32943603c4c5970.eot
fonts/fontawesome-webfont-a247f4358b253516c32943603c4c5970.eot.gz
fonts/fontawesome-webfont-f84ffa8dd9706d3cf7c8e800372c0cc0.woff
fonts/fontawesome-webfont.eot
fonts/fontawesome-webfont.eot.gz
fonts/fontawesome-webfont.svg
fonts/fontawesome-webfont.svg.gz
fonts/fontawesome-webfont.ttf
fonts/fontawesome-webfont.ttf.gz
fonts/fontawesome-webfont.woff
fonts/fontawesome-webfont.woff2
fonts/ionicons-05acfdb568b3df49ad31355b19495d4a.woff
fonts/ionicons-24712f6c47821394fba7942fbb52c3b2.ttf
fonts/ionicons-24712f6c47821394fba7942fbb52c3b2.ttf.gz
fonts/ionicons-2c2ae068be3b089e0a5b59abb1831550.eot
fonts/ionicons-2c2ae068be3b089e0a5b59abb1831550.eot.gz
fonts/ionicons-621bd386841f74e0053cb8e67f8a0604.svg
fonts/ionicons-621bd386841f74e0053cb8e67f8a0604.svg.gz
fonts/ionicons.eot
fonts/ionicons.eot.gz
fonts/ionicons.svg
fonts/ionicons.svg.gz
fonts/ionicons.ttf
fonts/ionicons.ttf.gz
fonts/ionicons.woff
images/
images/338-logo-0fd0fd60144901abfe725dcab3f8fde2.svg
images/338-logo-0fd0fd60144901abfe725dcab3f8fde2.svg.gz
images/338-logo.svg
images/338-logo.svg.gz
images/ex_admin/
images/ex_admin/admin_notes_icon-f8136850fced5cab46d5a5a6e9e6c5f5.png
images/ex_admin/admin_notes_icon.png
images/ex_admin/glyphicons-halflings-2516339970d710819585f90773aebe0a.png
images/ex_admin/glyphicons-halflings-white-9bbc6e9602998a385c2ea13df56470fd.png
images/ex_admin/glyphicons-halflings-white.png
images/ex_admin/glyphicons-halflings.png
images/ex_admin/orderable-331386b9e797ace4f53741fe9571fcea.png
images/ex_admin/orderable.png
images/ex_admin/datepicker/
images/ex_admin/datepicker/datepicker-header-bg-4042c3790975385f9f8ca85d26ff50e0.png
images/ex_admin/datepicker/datepicker-header-bg.png
images/ex_admin/datepicker/datepicker-input-icon-bc5c94af1483bafea6371c8ba726a55c.png
images/ex_admin/datepicker/datepicker-input-icon.png
images/ex_admin/datepicker/datepicker-next-link-icon-b3dd56edf2dc07858185f670ac03feaa.png
images/ex_admin/datepicker/datepicker-next-link-icon.png
images/ex_admin/datepicker/datepicker-nipple-653c3148630604936876361b26b2ef0b.png
images/ex_admin/datepicker/datepicker-nipple.png
images/ex_admin/datepicker/datepicker-prev-link-icon-1c8adbf6accf537be67f5a65017d285c.png
images/ex_admin/datepicker/datepicker-prev-link-icon.png
js/
js/admin_lte2-4909cedb51187283dde0004ff15fdccc.js
js/admin_lte2-4909cedb51187283dde0004ff15fdccc.js.gz
js/admin_lte2.js
js/admin_lte2.js-941a3d4bbdc82e2839bf37a1c539056a.map
js/admin_lte2.js.gz
js/admin_lte2.js.map
js/app-c8749b4babd4334991abb59e645bdab5.js
js/app-c8749b4babd4334991abb59e645bdab5.js.gz
js/app.js
js/app.js.gz
js/ex_admin_common-e2d9c0427496794c52e959858dcea2db.js
js/ex_admin_common-e2d9c0427496794c52e959858dcea2db.js.gz
js/ex_admin_common.js
js/ex_admin_common.js-b3d139445c6fe9ca20013bfa1179bf12.map
js/ex_admin_common.js.gz
js/ex_admin_common.js.map
js/jquery.min-5dec8239685b46c9621c96fa50cfd454.js
js/jquery.min-5dec8239685b46c9621c96fa50cfd454.js.gz
js/jquery.min.js
js/jquery.min.js-782053c400913f9b54e285db8ce6e997.map
js/jquery.min.js.gz
js/jquery.min.js.map
sent 12,816,180 bytes  received 2,670 bytes  25,637,700.00 bytes/sec
total size is 12,802,422  speedup is 1.00
       Running default compile
       
       > @ deploy /tmp/build_d33cbff6a12717d68c9fcd152d781478/assets
       > webpack --mode production
       
       Browserslist: caniuse-lite is outdated. Please run next command `npm update`
       Hash: 92747ff8751d512e65b1
       Version: webpack 4.41.0
       Time: 27047ms
       Built at: 05/03/2020 7:30:57 PM
       1 asset
       Entrypoint ./js/app.js = app.js
       [1] (webpack)/buildin/global.js 472 bytes {0} [built]
       [133] multi ./js/app.js 28 bytes {0} [built]
       [134] ./css/app.scss 3.95 KiB {0} [built] [failed] [1 error]
       [141] ./js/header.js 1.1 KiB {0} [built]
       [142] ./js/filter_players.js 721 bytes {0} [built]
       [143] ./js/filter_trade_form.js 1.48 KiB {0} [built]
       [144] ./js/filter_players_list.js 642 bytes {0} [built]
       [145] ./js/confirm_submit.js 270 bytes {0} [built]
       [146] (webpack)/buildin/module.js 497 bytes {0} [built]
       [147] ./node_modules/moment/locale sync ^./.*$ 3 KiB {0} [optional] [built]
       [148] ./js/app.js + 1 modules 2.27 KiB {0} [built]
       | ./js/app.js 1.54 KiB [built]
       | ./js/standings_chart.js 667 bytes [built]
       + 138 hidden modules
       
       ERROR in ./css/app.scss
       Module build failed (from ./node_modules/mini-css-extract-plugin/dist/loader.js):
       ModuleNotFoundError: Module not found: Error: Can't resolve '../../deps/phoenix_live_view/assets/css/live_view.css' in '/tmp/build_d33cbff6a12717d68c9fcd152d781478/assets/css'
       at factory.create (/tmp/build_d33cbff6a12717d68c9fcd152d781478/assets/node_modules/webpack/lib/Compilation.js:925:10)
       at factory (/tmp/build_d33cbff6a12717d68c9fcd152d781478/assets/node_modules/webpack/lib/NormalModuleFactory.js:401:22)
       at resolver (/tmp/build_d33cbff6a12717d68c9fcd152d781478/assets/node_modules/webpack/lib/NormalModuleFactory.js:130:21)
       at asyncLib.parallel.e (/tmp/build_d33cbff6a12717d68c9fcd152d781478/assets/node_modules/webpack/lib/NormalModuleFactory.js:224:22)
       at /tmp/build_d33cbff6a12717d68c9fcd152d781478/assets/node_modules/neo-async/async.js:2830:7
       at /tmp/build_d33cbff6a12717d68c9fcd152d781478/assets/node_modules/neo-async/async.js:6877:13
       at normalResolver.resolve (/tmp/build_d33cbff6a12717d68c9fcd152d781478/assets/node_modules/webpack/lib/NormalModuleFactory.js:214:25)
       at doResolve (/tmp/build_d33cbff6a12717d68c9fcd152d781478/assets/node_modules/enhanced-resolve/lib/Resolver.js:184:12)
       at hook.callAsync (/tmp/build_d33cbff6a12717d68c9fcd152d781478/assets/node_modules/enhanced-resolve/lib/Resolver.js:238:5)
       at _fn0 (eval at create (/tmp/build_d33cbff6a12717d68c9fcd152d781478/assets/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:15:1)
       at resolver.doResolve (/tmp/build_d33cbff6a12717d68c9fcd152d781478/assets/node_modules/enhanced-resolve/lib/UnsafeCachePlugin.js:37:5)
       at hook.callAsync (/tmp/build_d33cbff6a12717d68c9fcd152d781478/assets/node_modules/enhanced-resolve/lib/Resolver.js:238:5)
       at _fn0 (eval at create (/tmp/build_d33cbff6a12717d68c9fcd152d781478/assets/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:15:1)
       at hook.callAsync (/tmp/build_d33cbff6a12717d68c9fcd152d781478/assets/node_modules/enhanced-resolve/lib/Resolver.js:238:5)
       at _fn0 (eval at create (/tmp/build_d33cbff6a12717d68c9fcd152d781478/assets/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:27:1)
       at resolver.doResolve (/tmp/build_d33cbff6a12717d68c9fcd152d781478/assets/node_modules/enhanced-resolve/lib/DescriptionFilePlugin.js:42:38)
       at hook.callAsync (/tmp/build_d33cbff6a12717d68c9fcd152d781478/assets/node_modules/enhanced-resolve/lib/Resolver.js:238:5)
       at _fn42 (eval at create (/tmp/build_d33cbff6a12717d68c9fcd152d781478/assets/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:16:1)
       at hook.callAsync (/tmp/build_d33cbff6a12717d68c9fcd152d781478/assets/node_modules/enhanced-resolve/lib/Resolver.js:238:5)
       at _fn0 (eval at create (/tmp/build_d33cbff6a12717d68c9fcd152d781478/assets/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:27:1)
       at resolver.doResolve (/tmp/build_d33cbff6a12717d68c9fcd152d781478/assets/node_modules/enhanced-resolve/lib/DescriptionFilePlugin.js:42:38)
       at hook.callAsync (/tmp/build_d33cbff6a12717d68c9fcd152d781478/assets/node_modules/enhanced-resolve/lib/Resolver.js:238:5)
       at _fn1 (eval at create (/tmp/build_d33cbff6a12717d68c9fcd152d781478/assets/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:16:1)
       at hook.callAsync (/tmp/build_d33cbff6a12717d68c9fcd152d781478/assets/node_modules/enhanced-resolve/lib/Resolver.js:238:5)
       at _fn0 (eval at create (/tmp/build_d33cbff6a12717d68c9fcd152d781478/assets/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:15:1)
       at fs.stat (/tmp/build_d33cbff6a12717d68c9fcd152d781478/assets/node_modules/enhanced-resolve/lib/DirectoryExistsPlugin.js:22:13)
       at process.nextTick (/tmp/build_d33cbff6a12717d68c9fcd152d781478/assets/node_modules/enhanced-resolve/lib/CachedInputFileSystem.js:73:15)
       at _combinedTickCallback (internal/process/next_tick.js:67:7)
       at process._tickCallback (internal/process/next_tick.js:98:9)
       @ ./js/app.js 4:0-34
       @ multi ./js/app.js
       Child mini-css-extract-plugin node_modules/css-loader/index.js!node_modules/sass-loader/lib/loader.js??ref--6-2!css/app.scss:
       Entrypoint mini-css-extract-plugin = *
       [0] ./node_modules/css-loader!./node_modules/sass-loader/lib/loader.js??ref--6-2!./css/app.scss 27.6 KiB {0} [built]
       + 1 hidden module
       
       ERROR in ./css/app.scss (./node_modules/css-loader!./node_modules/sass-loader/lib/loader.js??ref--6-2!./css/app.scss)
       Module not found: Error: Can't resolve '../../deps/phoenix_live_view/assets/css/live_view.css' in '/tmp/build_d33cbff6a12717d68c9fcd152d781478/assets/css'
       @ ./css/app.scss (./node_modules/css-loader!./node_modules/sass-loader/lib/loader.js??ref--6-2!./css/app.scss) 3:10-112
       
       npm ERR! Linux 4.4.0-1062-aws
       npm ERR! argv "/tmp/build_d33cbff6a12717d68c9fcd152d781478/.heroku/node/bin/node" "/tmp/build_d33cbff6a12717d68c9fcd152d781478/.heroku/node/bin/npm" "run" "deploy"
       npm ERR! node v6.9.2
       npm ERR! npm  v3.10.9
       npm ERR! code ELIFECYCLE
       npm ERR! @ deploy: `webpack --mode production`
       npm ERR! Exit status 2
       npm ERR!
       npm ERR! Failed at the @ deploy script 'webpack --mode production'.
       npm ERR! Make sure you have the latest version of node.js and npm installed.
       npm ERR! If you do, this is most likely a problem with the  package,
       npm ERR! not with npm itself.
       npm ERR! Tell the author that this fails on your system:
       npm ERR!     webpack --mode production
       npm ERR! You can get information on how to open an issue for this project with:
       npm ERR!     npm bugs
       npm ERR! Or if that isn't available, you can get their info via:
       npm ERR!     npm owner ls
       npm ERR! There is likely additional logging output above.
       
       npm ERR! Please include the following file with any support request:
       npm ERR!     /tmp/build_d33cbff6a12717d68c9fcd152d781478/assets/npm-debug.log
 !     Push rejected, failed to compile Phoenix app.
 !     Push failed
```

--- a/assets/css/app.scss
+++ b/assets/css/app.scss
@@ -1,7 +1,6 @@
 /* This file is for your main application css. */
 @import 'bourbon';
 @import 'neat';
-@import "../../deps/phoenix_live_view/assets/css/live_view.css";
 
 @import 'base/base';
 @import 'layout/base';
@@ -21,3 +20,63 @@
 @import 'owners';
 @import 'trades';
 @import 'pages';
+
+@import "../node_modules/nprogress/nprogress.css";
+
+/* LiveView specific classes for your customizations */
+.invalid-feedback {
+  color: #a94442;
+  display: block;
+  margin: -1rem 0 2rem;
+}
+
+.phx-no-feedback.invalid-feedback, .phx-no-feedback .invalid-feedback {
+  display: none;
+}
+
+.phx-click-loading {
+  opacity: 0.5;
+  transition: opacity 1s ease-out;
+}
+
+.phx-disconnected{
+  cursor: wait;
+}
+.phx-disconnected *{
+  pointer-events: none;
+}
+
+.phx-modal {
+  opacity: 1!important;
+  position: fixed;
+  z-index: 1;
+  left: 0;
+  top: 0;
+  width: 100%;
+  height: 100%;
+  overflow: auto;
+  background-color: rgb(0,0,0);
+  background-color: rgba(0,0,0,0.4);
+}
+
+.phx-modal-content {
+  background-color: #fefefe;
+  margin: 15% auto;
+  padding: 20px;
+  border: 1px solid #888;
+  width: 80%;
+}
+
+.phx-modal-close {
+  color: #aaa;
+  float: right;
+  font-size: 28px;
+  font-weight: bold;
+}
+
+.phx-modal-close:hover,
+.phx-modal-close:focus {
+  color: black;
+  text-decoration: none;
+  cursor: pointer;
+}

--- a/assets/js/app.js
+++ b/assets/js/app.js
@@ -11,6 +11,7 @@ import css from "../css/app.scss";
 //
 import "phoenix_html";
 import { Socket } from "phoenix";
+import NProgress from "nprogress";
 import LiveSocket from "phoenix_live_view";
 
 // LiveView polyfills for IE11
@@ -37,6 +38,10 @@ let csrfToken = document
 let liveSocket = new LiveSocket("/live", Socket, {
   params: { _csrf_token: csrfToken }
 });
+
+// Show progress bar on live navigation and form submits
+window.addEventListener("phx:page-loading-start", info => NProgress.start());
+window.addEventListener("phx:page-loading-stop", info => NProgress.done());
 
 // connect if there are any LiveViews on the page
 liveSocket.connect();

--- a/assets/package.json
+++ b/assets/package.json
@@ -12,6 +12,7 @@
     "formdata-polyfill": "^3.0.18",
     "jquery": "^3.1.0",
     "mdn-polyfills": "^5.17.0",
+    "nprogress": "^0.2.0",
     "phoenix": "file:../deps/phoenix",
     "phoenix_html": "file:../deps/phoenix_html",
     "phoenix_live_view": "file:../deps/phoenix_live_view",

